src/schema/__tests__/partner_show.test.js

```diff
@@ -233,7 +225,9 @@ describe("PartnerShow type", () => {
     })
   })
   it("includes the number of artworks by a specific artist", () => {
-    total.onCall(0).returns(Promise.resolve(2))
+    rootValue.partnerShowArtworksLoader = sinon
+      .stub()
+      .returns(Promise.resolve({ headers: { "x-total-count": 2 } }))
```

Comment:
I’d also suggest getting rid of sinon over builtin jest stubs.

Reply:
👍 can I tackle that in a separate issue/cleanup? There are...a lot of those :)

Reply:
I would recommend it, there's a tool ([jest-codemods](https://github.com/skovhus/jest-codemods)) that might help here

Reply:
It's a WIP, https://github.com/skovhus/jest-codemods/issues/68#issuecomment-363054214

Reply:
Oh absolutely, it was only an observation.